File: WatchWhat.GraphQL/Shared/tmdb/dtos/video.ts
export type Videos = {
  id: number,
  results: Video[]
};

export type Video = {
  id: string,
  iso_639_1: string,
  iso_3166_1: string,
  key: string,
  name: string,
  official: boolean,
  published_at: Date,
  site: string,
  size: number,
  type: string
};

File: WatchWhat.Apple/Shared/Models/Video.swift
//
//  Video.swift
//  WatchWhat
//
//  Created by Drew Miller on 3/18/23.
//

import Foundation

struct VideoPage {
    let page: Int
    let total_pages: Int
    let total_results: Int
    let results: [Video]
}

struct Video {
    let id: String
    let key: String
    let name: String?
    let official: Bool
    let publishedAt: String
    let site: String
    let size: Int
    let type: String
}
